msg/en_US.utf8/csql.msg

```diff
@@ -188,6 +190,12 @@ $ csql help messages
    ;HISTORYRead <history_num>   - read entry on the history number into command buffer.\n\
    ;TRAce [ON/OFF] [text/json]  - enable/disable sql auto trace.\n\
    ;SIngleline [ON|OFF]         - enable/disable single-line mode.\n\
+   ;Connect username [dbname | dbname@hostname]\n\
```

Comment:
How about changing to "Connect to the current or another database as a user." ?

Reply:
I see, How about "connects a given username to the database."??

I referenced other DB's manual.

Reply:
LGTM